Containers/mastercontainer/Dockerfile

```diff
@@ -5,7 +5,7 @@ FROM docker:20.10.21-dind-alpine3.16 as dind
 FROM caddy:2.6.2-alpine as caddy
 
 # From https://github.com/docker-library/php/blob/master/8.0/bullseye/apache/Dockerfile
-FROM php:8.0.26-apache-bullseye
+FROM alpine:3.16.3
```

Comment:
Thanks! But can we maybe use the php-alpine image for this or is this not possible?

Reply:
I think it isn't possible or very hard to integrate, this is the easiest way I see
Maybe we could use the php:fpm-alpine image and run fpm using supervisor

Reply:
> I think it isn't possible or very hard to integrate, this is the easiest way I see

Ah I see.

> Maybe we could use the php:fpm-alpine image and run fpm using supervisor

Yes, I would prefer this over building from alpine directly

Reply:
@szaimen do you think this could work?

Reply:
Yes, looks pretty good to me already! Thanks a lot! :)
I will also include this in the release after 4.0.1 :)